### PR TITLE
Android App that uses Extole SDK

### PR DESCRIPTION
**Pull request by daniftodi**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        maven {
+            url "http://artifactory.tools.intole.net/artifactory/extole-packages/api-stubs"
+            allowInsecureProtocol true
+        }
+        maven {
+            url "http://artifactory.tools.intole.net/artifactory/extole-packages/"
```
Comment (mcyster) [inline settings.gradle:13]: Please add a TODO with ticket number as clients can't use this

and abvobe

File: app/src/main/java/com/extole/androidsdk/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        val nativeShare: Button = findViewById(R.id.native_share)
+        nativeShare.setOnClickListener {
+            extole.nativeShare(
+                extole.getText("sharing.email.subject"),
+                extole.getText("sharing.email.message") + " " + extole.getText("me.shareable_link")
+            )
+        }
+    }
+
+    fun loadCreative(extole: Extole) {
```
Comment (mcyster) [inline app/src/main/java/com/extole/androidsdk/MainActivity.kt]: loadShareExperience?

**Review by mcyster — APPROVED** [review]
:+1: minor comments

File: settings.gradle (hunk 1 above)
Comment (daniftodi) [reply settings.gradle:13]: added: `// TODO to be replaced by Github Packages ENG-15109`

File: app/src/main/java/com/extole/androidsdk/MainActivity.kt (hunk 2 above)
Comment (daniftodi) [reply app/src/main/java/com/extole/androidsdk/MainActivity.kt]: renamed

**Pull request merged**